app/com/linkedin/drelephant/schedulers/AirflowScheduler.java

```diff
@@ -58,10 +62,14 @@ public AirflowScheduler(String appId, Properties properties, SchedulerConfigurat
   }
 
   private void loadInfo(String appId, Properties properties) {
-    // Update the 4 Ids
+    // examples:
+    // my_amazing_task_id
     _taskId = properties.getProperty(AIRFLOW_TASK_ID);
+    // 2016-06-27T01:30:00
```

Comment:
Cool, thanks for the clarification. Just one more doubt. What is the upper limit on the length of _dagId and _taskId?